src/provider/controller/ResolverHandler.ts

```diff
@@ -18,7 +18,7 @@ const RESOLVER_URL = (() => {
     }
 
     // Locate the default resolver HTML page, relative to the location of the provider
-    return providerLocation.replace('provider.html', 'ui/resolver');
+    return providerLocation.replace('provider.html', 'ui/resolver/index.html');
```

Comment:
Maybe a silly question, but do we not have `index.*` be the default document for the server?